src/applications/personalization/dashboard/containers/MessagingWidget.jsx

```diff
@@ -35,7 +35,7 @@ class MessagingWidget extends React.Component {
       { label: 'Date', value: 'sentDate', nonSortable: true },
     ];
 
-    // eslint-disable-next-line
+    // eslint-disable-next-line no-unused-vars
     const makeMessageLink = (content, id) => (
```

Comment:
Some refactoring could be done here rather than disabling the rule.

Reply:
Doesn't prefixing with an underscore allow the variable? `_id` = eslint should ignore it because it's a valid placeholder